### PR TITLE
3.14.0 Changelog

### PR DESCRIPTION
**Pull request by chenshuai2144**
First of all, thank you for your contribution! 😄

New feature please send pull request to feature branch, and rest to master branch.
Pull request will be merged after one of collaborators approve.
Please makes sure that these form are filled before submitting your pull request, thank you!

[[中文版模板 / Chinese template](https://github.com/ant-design/ant-design/blob/master/.github/PULL_REQUEST_TEMPLATE/pr_cn.md)]

### This is a ...

- [ ] New feature
- [ ] Bug fix
- [x] Site / document update
- [ ] Component style update
- [ ] TypeScript definition update
- [ ] Refactoring
- [ ] Code style optimization
- [ ] Branch merge
- [ ] Other (about what?)

### What's the background?

> 1. Describe the source of requirement.
> 2. Resolve what problem.
> 3. Related issue link.

### API Realization (Optional if not new feature)

> 1. Basic thought of solution and other optional proposal.
> 2. List final API realization and usage sample.
> 3. GIF or snapshot should be provided if includes UI/interactive modification.

### What's the effect? (Optional if not new feature)

> 1. Does this PR affect user? Which part will be affected?
> 2. What will say in changelog?
> 3. Does this PR contains potential break change or other risk?

### Changelog description (Optional if not new feature)

> 1. English description
> 2. Chinese description (optional)

### Self Check before Merge

- [ ] Doc is updated/provided or not needed
- [ ] Demo is updated/provided or not needed
- [ ] TypeScript definition is updated/provided or not needed
- [ ] Changelog is provided or not needed

### Additional Plan? (Optional if not new feature)

> If this PR related with other PR or following info. You can type here.

### CODE REVIEW

File: CHANGELOG.zh-CN.md [hunk 1]
```diff
@@ -15,6 +15,29 @@ timeline: true
...
+- 🌟 Countdown 新增了 `onFinish` prop。[#14791](https://github.com/ant-design/ant-design/pull/14791)
+- 🌟 Collapse.Panel 新增了 `extra`。[62e65d](https://github.com/ant-design/ant-design/commit/62e65d955065b1862240f9f30d84de44349a0cf9)
+- 🌟 Pagination 支持通过 `showQuickJumper` 自定义分页跳转按钮。 [#14798](https://github.com/ant-design/ant-design/issues/14798)
+- 🐞 订正了 Table `column filter` 的 typescript 类型问题。[#14777](https://github.com/ant-design/ant-design/issues/14777)
+- DatePicker
+  - 🐞 修复 DatePicker `name` prop 无效的问题。[#15029](https://github.com/ant-design/ant-design/pull/15029)
+  - 🐞 修复 DatePicker `onPanelChange`  typescript 声明缺失的问题。[#15043](https://github.com/ant-design/ant-design/pull/15043)
+  - 🌟 DatePicker 支持 `separator` prop。[#15055](https://github.com/ant-design/ant-design/pull/15055)
+- 🌟 Form 支持 labelCol & wrapperCol prop。[#15038](https://github.com/ant-design/ant-design/pull/15038)
+- 🌟 Icon 增加了 'more' 的图标。[#15047](https://github.com/ant-design/ant-design/pull/15047)
```
Comment (zy410419243) [inline CHANGELOG.zh-CN.md]: `more`

Comment (zy410419243) [reply CHANGELOG.zh-CN.md]: 好像也不对，要么去掉单引号...

File: CHANGELOG.zh-CN.md [hunk 2]
```diff
@@ -15,6 +15,29 @@ timeline: true
...
+`2019-03-02`
+
+- 本月新增了两个组件：
+  - 🔥🔥🔥 [Typography](https://github.com/ant-design/ant-design/pull/14250) 提供了文本的基本格式及常见操作。
+  - 🔥🔥🔥 [PageHeader](https://github.com/ant-design/ant-design/pull/13637) 可用于声明页面主题、展示用户所关注的页面重要信息，以及承载与当前页相关的操作项。
+- 🌟 Timepicker 新增了 `clearIcon` prop，用于自定义清除图标。[#14556](https://github.com/ant-design/ant-design/pull/14556)
+- 🌟 Countdown 新增了 `onFinish` prop。[#14791](https://github.com/ant-design/ant-design/pull/14791)
+- 🌟 Collapse.Panel 新增了 `extra`。[62e65d](https://github.com/ant-design/ant-design/commit/62e65d955065b1862240f9f30d84de44349a0cf9)
+- 🌟 Pagination 支持通过 `showQuickJumper` 自定义分页跳转按钮。 [#14798](https://github.com/ant-design/ant-design/issues/14798)
+- 🐞 订正了 Table `column filter` 的 typescript 类型问题。[#14777](https://github.com/ant-design/ant-design/issues/14777)
```
Comment (zy410419243) [inline CHANGELOG.zh-CN.md]: 照以往的日志看，TypeScript 类型修复都是放在最后讲，而且格式似乎都固定了的

File: CHANGELOG.en-US.md [hunk 3]
```diff
@@ -14,6 +14,28 @@ timeline: true
 * Major version release is not included in this schedule for breaking change and new features.
 
 ---
+## 3.14.0
+
+`2019-03-02`
+
+- Two new components added this month:
```
Comment (zombieJ) [inline CHANGELOG.en-US.md]: 参考一下以前的写法：https://github.com/ant-design/ant-design/releases/tag/3.13.0

File: CHANGELOG.zh-CN.md [hunk 4]
```diff
@@ -15,6 +15,29 @@ timeline: true
...
+
+`2019-03-02`
+
+- 本月新增了两个组件：
+  - 🔥🔥🔥 [Typography](https://github.com/ant-design/ant-design/pull/14250) 提供了文本的基本格式及常见操作。
+  - 🔥🔥🔥 [PageHeader](https://github.com/ant-design/ant-design/pull/13637) 可用于声明页面主题、展示用户所关注的页面重要信息，以及承载与当前页相关的操作项。
+- 🌟 Timepicker 新增了 `clearIcon` prop，用于自定义清除图标。[#14556](https://github.com/ant-design/ant-design/pull/14556)
+- 🌟 Countdown 新增了 `onFinish` prop。[#14791](https://github.com/ant-design/ant-design/pull/14791)
+- 🌟 Collapse.Panel 新增了 `extra`。[62e65d](https://github.com/ant-design/ant-design/commit/62e65d955065b1862240f9f30d84de44349a0cf9)
+- 🌟 Pagination 支持通过 `showQuickJumper` 自定义分页跳转按钮。 [#14798](https://github.com/ant-design/ant-design/issues/14798)
```
Comment (zy410419243) [inline CHANGELOG.zh-CN.md]: 其他都是 pr，就这个是 issue

https://github.com/ant-design/ant-design/pull/14819

File: CHANGELOG.en-US.md [hunk 5]
```diff
@@ -14,6 +14,28 @@ timeline: true
...
+  - 🔥🔥🔥[Typography](https://github.com/ant-design/ant-design/pull/14250) provides basic formatting and common operations for text.
+  - 🔥🔥🔥[PageHeader](https://github.com/ant-design/ant-design/pull/13637) can be used to declare the page theme, display important information about the page that the user is interested in, and host the relevant page. Action item.
+- 🌟Timepicker added `clearIcon` prop for custom clearing icons. [#14556](https://github.com/ant-design/ant-design/pull/14556)
```
Comment (zombieJ) [inline CHANGELOG.en-US.md]: added -> provides, custom -> customizing, clearing -> clear

File: CHANGELOG.en-US.md [hunk 6]
```diff
@@ -14,6 +14,28 @@ timeline: true
...
+- 🌟Added `onFinish` prop to the countdown. [#14791](https://github.com/ant-design/ant-design/pull/14791)
```
Comment (zombieJ) [inline CHANGELOG.en-US.md]: 你看一下 Changelog description，里面有翻译好的描述

File: CHANGELOG.zh-CN.md [hunk 7]
```diff
@@ -15,6 +15,29 @@ timeline: true
...
+- 🐞 订正了 Table `column filter` 的 typescript 类型问题。[#14777](https://github.com/ant-design/ant-design/issues/14777)
+- DatePicker
+  - 🐞 修复 DatePicker `name` prop 无效的问题。[#15029](https://github.com/ant-design/ant-design/pull/15029)
+  - 🐞 修复 DatePicker `onPanelChange`  typescript 声明缺失的问题。[#15043](https://github.com/ant-design/ant-design/pull/15043)
+  - 🌟 DatePicker 支持 `separator` prop。[#15055](https://github.com/ant-design/ant-design/pull/15055)
+- 🌟 Form 支持 labelCol & wrapperCol prop。[#15038](https://github.com/ant-design/ant-design/pull/15038)
```
Comment (zy410419243) [inline CHANGELOG.zh-CN.md]: 属性似乎要用 ` 包着

File: CHANGELOG.zh-CN.md [hunk 8]
```diff
@@ -15,6 +15,29 @@ timeline: true
...
+- 🌟 Icon 增加了 'more' 的图标。[#15047](https://github.com/ant-design/ant-design/pull/15047)
+- 🐞 修复 Table 筛选不支持 string 以外类型的问题。[#15046](https://github.com/ant-design/ant-design/pull/15046)
+- 🐞 修复 Spin `wrapperClassName` 设置 `padding` 图标不居中的问题。[#13648](https://github.com/ant-design/ant-design/issues/13648)
```
Comment (zy410419243) [inline CHANGELOG.zh-CN.md]: https://github.com/ant-design/ant-design/pull/15056

File: CHANGELOG.en-US.md (hunk 3 above)
Comment (chenshuai2144) [reply CHANGELOG.en-US.md]: 我感觉这样好一点

File: CHANGELOG.en-US.md (hunk 5 above)
Comment (ztplz) [inline CHANGELOG.en-US.md]: `Timepicker` => `TimePicker`

**Review by ztplz — DISMISSED** [review]

File: CHANGELOG.zh-CN.md [hunk 9]
```diff
@@ -15,6 +15,29 @@ timeline: true
...
 
+## 3.14.0
+
+`2019-03-02`
+
+- 本月新增了两个组件：
+  - 🔥🔥🔥 [Typography](https://github.com/ant-design/ant-design/pull/14250) 提供了文本的基本格式及常见操作。
+  - 🔥🔥🔥 [PageHeader](https://github.com/ant-design/ant-design/pull/13637) 可用于声明页面主题、展示用户所关注的页面重要信息，以及承载与当前页相关的操作项。
+- 🌟 TimePicker 新增了 `clearIcon` prop，用于自定义清除图标。[#14556](https://github.com/ant-design/ant-design/pull/14556)
+- 🌟Statistic.Countdown 支持 onFinish prop。[#14791](https://github.com/ant-design/ant-design/pull/14791)
```
Comment (zy410419243) [inline CHANGELOG.zh-CN.md]: `onFinish`

File: CHANGELOG.en-US.md [hunk 10]
```diff
@@ -14,6 +14,28 @@ timeline: true
...
+- Two new components added this month:
+  - 🔥🔥🔥[Typography](https://github.com/ant-design/ant-design/pull/14250) provides basic formatting and common operations for text.
+  - 🔥🔥🔥[PageHeader](https://github.com/ant-design/ant-design/pull/13637) can be used to declare the page theme, display important information about the page that the user is interested in, and host the relevant page. Action item.
+- 🌟 TimePicker provides `clearIcon` prop for customizing clear icons. [#14556](https://github.com/ant-design/ant-design/pull/14556)
+- 🌟 Statistic.Countdown support onFinish prop. [#14791](https://github.com/ant-design/ant-design/pull/14791)
+- 🌟 Support `extra` for Collapse.Panel. [62e65d](https://github.com/ant-design/ant-design/commit/62e65d955065b1862240f9f30d84de44349a0cf9)
+- 🌟 Support goto button in Pagination.  [#14798](https://github.com/ant-design/ant-design/pull/14819)
+- 🐞 Fixed the typescript type problem for Table`Column Filter`. [#15056](https://github.com/ant-design/ant-design/pull/15056)
+- DatePicker
+  - 🐞 Fix DatePicker not suppor name prop. [#15029](https://github.com/ant-design/ant-design/pull/15029)
```
Comment (zy410419243) [inline CHANGELOG.en-US.md]: support

File: CHANGELOG.en-US.md [hunk 11]
```diff
@@ -14,6 +14,28 @@ timeline: true
...
 ---
+## 3.14.0
+
+`2019-03-02`
+
+- Two new components added this month:
+  - 🔥🔥🔥[Typography](https://github.com/ant-design/ant-design/pull/14250) provides basic formatting and common operations for text.
+  - 🔥🔥🔥[PageHeader](https://github.com/ant-design/ant-design/pull/13637) can be used to declare the page theme, display important information about the page that the user is interested in, and host the relevant page. Action item.
+- 🌟 TimePicker provides `clearIcon` prop for customizing clear icons. [#14556](https://github.com/ant-design/ant-design/pull/14556)
+- 🌟 Statistic.Countdown support onFinish prop. [#14791](https://github.com/ant-design/ant-design/pull/14791)
```
Comment (zy410419243) [inline CHANGELOG.en-US.md]: `onFinish`

File: CHANGELOG.en-US.md [hunk 12]
```diff
@@ -14,6 +14,28 @@ timeline: true
...
+- 🌟 Support `extra` for Collapse.Panel. [62e65d](https://github.com/ant-design/ant-design/commit/62e65d955065b1862240f9f30d84de44349a0cf9)
+- 🌟 Support goto button in Pagination.  [#14798](https://github.com/ant-design/ant-design/pull/14819)
+- 🐞 Fixed the typescript type problem for Table`Column Filter`. [#15056](https://github.com/ant-design/ant-design/pull/15056)
```
Comment (zy410419243) [inline CHANGELOG.en-US.md]: 1. Fixed => Fix
2. Table 后面少空格

File: CHANGELOG.en-US.md [hunk 13]
```diff
@@ -14,6 +14,28 @@ timeline: true
...
+- DatePicker
+  - 🐞 Fix DatePicker not suppor name prop. [#15029](https://github.com/ant-design/ant-design/pull/15029)
+  - 🐞 Fix missing problems with DatePicker`onPanelChange` typescript. [#15043](https://github.com/ant-design/ant-design/pull/15043)
+  - 🌟 DatePicker supports `separator` prop. [#15055](https://github.com/ant-design/ant-design/pull/15055)
+- 🌟 The table supports `labelCol` & `wrapperCol` prop. [#15038](https://github.com/ant-design/ant-design/pull/15038)
```
Comment (zy410419243) [inline CHANGELOG.en-US.md]: The table => `Form`

File: CHANGELOG.en-US.md [hunk 14]
```diff
@@ -14,6 +14,28 @@ timeline: true
...
+- 🌟 The icon for `more` has been added to the icon. [#15047](https://github.com/ant-design/ant-design/pull/15047)
```
Comment (zy410419243) [inline CHANGELOG.en-US.md]: The icon => `Icon`

Comment (zy410419243) [reply CHANGELOG.en-US.md]: 或者改下句式
Add type `more` for Icon.

File: CHANGELOG.en-US.md [hunk 15]
```diff
@@ -14,6 +14,28 @@ timeline: true
...
+- 🐞 Fix Table filter can not support other type of value. [#15046](https://github.com/ant-design/ant-design/pull/15046)
+- 🐞 Fix Spin `wrapperClassName` setting `padding` icon is not centered. [#15056](https://github.com/ant-design/ant-design/pull/15056)
+- 🐞 Fix calendar `onPanelChange` in some cases will not trigger the problem. [#15063](https://github.com/ant-design/ant-design/pull/15063)
```
Comment (zy410419243) [inline CHANGELOG.en-US.md]: `Calendar`

**Comment by zy410419243** [comment]
~~我好无聊啊~~

File: CHANGELOG.en-US.md (hunk 15 above)
Comment (zy410419243) [reply CHANGELOG.en-US.md]: wil not be triggered.

讲起来不会触发指的是不会触发啥？

File: CHANGELOG.en-US.md [hunk 16]
```diff
@@ -14,6 +14,28 @@ timeline: true
...
 ---
+## 3.14.0
+
+`2019-03-02`
+
+- Two new components added this month:
+  - 🔥🔥🔥[Typography](https://github.com/ant-design/ant-design/pull/14250) provides basic formatting and common operations for text.
+  - 🔥🔥🔥[PageHeader](https://github.com/ant-design/ant-design/pull/13637) can be used to declare the page theme, display important information about the page that the user is interested in, and host the relevant page. Action item.
+- 🌟 TimePicker provides `clearIcon` prop for customizing clear icons. [#14556](https://github.com/ant-design/ant-design/pull/14556)
+- 🌟 Statistic.Countdown support `onFinish` prop. [#14791](https://github.com/ant-design/ant-design/pull/14791)
```
Comment (zy410419243) [inline CHANGELOG.en-US.md]: supports

File: CHANGELOG.en-US.md [hunk 17]
```diff
@@ -14,6 +14,28 @@ timeline: true
 * Major version release is not included in this schedule for breaking change and new features.
 
 ---
+## 3.14.0
+
+`2019-03-02`
+
+- Two new components added this month:
+  - 🔥🔥🔥[Typography](https://github.com/ant-design/ant-design/pull/14250) provides basic formatting and common operations for text.
+  - 🔥🔥🔥[PageHeader](https://github.com/ant-design/ant-design/pull/13637) can be used to declare the page theme, display important information about the page that the user is interested in, and host the relevant page. Action item.
```
Comment (zy410419243) [inline CHANGELOG.en-US.md]: 这 `Action item` 为啥是单独的...

File: CHANGELOG.en-US.md [hunk 18]
```diff
@@ -14,6 +14,28 @@ timeline: true
...
+- 🌟 TimePicker provides `clearIcon` prop for customizing clear icons. [#14556](https://github.com/ant-design/ant-design/pull/14556)
+- 🌟 Statistic.Countdown support `onFinish` prop. [#14791](https://github.com/ant-design/ant-design/pull/14791)
+- 🌟 Support `extra` for Collapse.Panel. [62e65d](https://github.com/ant-design/ant-design/commit/62e65d955065b1862240f9f30d84de44349a0cf9)
+- 🌟 Support goto button in Pagination.  [#14798](https://github.com/ant-design/ant-design/pull/14819)
+- 🐞 Fix the typescript type problem for Table `Column Filter`. [#15056](https://github.com/ant-design/ant-design/pull/15056)
+- DatePicker
+  - 🐞 Fix DatePicker not support name prop. [#15029](https://github.com/ant-design/ant-design/pull/15029)
+  - 🐞 Fix missing problems with DatePicker`onPanelChange` typescript. [#15043](https://github.com/ant-design/ant-design/pull/15043)
```
Comment (zy410419243) [inline CHANGELOG.en-US.md]: 属性前面的空格

File: CHANGELOG.en-US.md [hunk 19]
```diff
@@ -14,6 +14,28 @@ timeline: true
...
+  - 🌟 DatePicker supports `separator` prop. [#15055](https://github.com/ant-design/ant-design/pull/15055)
+- 🌟 The table supports `labelCol` & `wrapperCol` prop. [#15038](https://github.com/ant-design/ant-design/pull/15038)
+- 🌟 The icon for `more` has been added to the icon. [#15047](https://github.com/ant-design/ant-design/pull/15047)
+- 🐞 Fix Table filter can not support other type of value. [#15046](https://github.com/ant-design/ant-design/pull/15046)
+- 🐞 Fix Spin `wrapperClassName` setting `padding` icon is not centered. [#15056](https://github.com/ant-design/ant-design/pull/15056)
+- 🐞 Fix calendar `onPanelChange` in some cases will not trigger the problem. [#15063](https://github.com/ant-design/ant-design/pull/15063)
+- 🐞 Select to support `showArrow` in multi-select mode. [#15091](https://github.com/ant-design/ant-design/pull/15091)
```
Comment (zy410419243) [inline CHANGELOG.en-US.md]: supports

File: CHANGELOG.zh-CN.md [hunk 20]
```diff
@@ -15,6 +15,29 @@ timeline: true
...
+- DatePicker
+  - 🐞 修复 DatePicker `name` prop 无效的问题。[#15029](https://github.com/ant-design/ant-design/pull/15029)
+  - 🐞 修复 DatePicker `onPanelChange`  typescript 声明缺失的问题。[#15043](https://github.com/ant-design/ant-design/pull/15043)
+  - 🌟 DatePicker 支持 `separator` prop。[#15055](https://github.com/ant-design/ant-design/pull/15055)
+- 🌟 Form 支持 `labelCol` & `wrapperCol` prop。[#15038](https://github.com/ant-design/ant-design/pull/15038)
+- 🌟 Icon 增加了 `more` 的图标。[#15047](https://github.com/ant-design/ant-design/pull/15047)
+- 🐞 修复 Table 筛选不支持 string 以外类型的问题。[#15046](https://github.com/ant-design/ant-design/pull/15046)
+- 🐞 修复 Spin `wrapperClassName` 设置 `padding` 图标不居中的问题。[#15056](https://github.com/ant-design/ant-design/pull/15056)
+- 🐞 修复 Calendar `onPanelChange` 在某些情况下不会触发的问题。[#15063](https://github.com/ant-design/ant-design/pull/15063)
+- 🐞 Select 在多选模式下支持 `showArrow`。[#15091](https://github.com/ant-design/ant-design/pull/15091)
```
Comment (zy410419243) [inline CHANGELOG.zh-CN.md]: 这个不是 bug 吧

File: CHANGELOG.en-US.md [hunk 21]
```diff
@@ -14,6 +14,28 @@ timeline: true
...
+
+`2019-03-02`
+
+- Two new components added this month:
+  - 🔥🔥🔥[Typography](https://github.com/ant-design/ant-design/pull/14250) provides basic formatting and common operations for text.
+  - 🔥🔥🔥[PageHeader](https://github.com/ant-design/ant-design/pull/13637) can be used to declare the page theme, display important information about the page that the user is interested in, and host the relevant page. Action item.
+- 🌟 TimePicker provides `clearIcon` prop for customizing clear icons. [#14556](https://github.com/ant-design/ant-design/pull/14556)
+- 🌟 Statistic.Countdown support `onFinish` prop. [#14791](https://github.com/ant-design/ant-design/pull/14791)
+- 🌟 Support `extra` for Collapse.Panel. [62e65d](https://github.com/ant-design/ant-design/commit/62e65d955065b1862240f9f30d84de44349a0cf9)
+- 🌟 Support goto button in Pagination.  [#14798](https://github.com/ant-design/ant-design/pull/14819)
```
Comment (zy410419243) [inline CHANGELOG.en-US.md]: 链接文本跟链接没对上

**Comment by zy410419243** [comment]
u7ku4x9 u lub 1kr 89mm 好吧，饭吃完了，不知道还有没有没 review 出来的地方

File: CHANGELOG.en-US.md (hunk 15 above)
Comment (chenshuai2144) [reply CHANGELOG.en-US.md]: 不会触发 onPanelChange

**Comment by chenshuai2144** [comment]
...我的标准是在 google 翻译里面能翻译成流畅的中文

File: CHANGELOG.en-US.md (hunk 15 above)
Comment (zy410419243) [reply CHANGELOG.en-US.md]: Fix Calendar won't trigger `onPanelChange` correctly in some cases.

File: CHANGELOG.en-US.md [hunk 22]
```diff
@@ -14,6 +14,28 @@ timeline: true
...
+- DatePicker
+  - 🐞 Fix DatePicker not support name prop. [#15029](https://github.com/ant-design/ant-design/pull/15029)
+  - 🐞 Fix missing problems with DatePicker `onPanelChange` typescript. [#15043](https://github.com/ant-design/ant-design/pull/15043)
+  - 🌟 DatePicker supports `separator` prop. [#15055](https://github.com/ant-design/ant-design/pull/15055)
+- 🌟 The Form supports `labelCol` & `wrapperCol` prop. [#15038](https://github.com/ant-design/ant-design/pull/15038)
+- 🌟 Add type `more` for Icon. [#15047](https://github.com/ant-design/ant-design/pull/15047)
+- 🐞 Fix Table filter can not support other type of value. [#15046](https://github.com/ant-design/ant-design/pull/15046)
+- 🐞 Fix Spin `wrapperClassName` setting `padding` icon is not centered. [#15056](https://github.com/ant-design/ant-design/pull/15056)
+- 🐞 Fix calendar `onPanelChange` in some cases will not trigger the problem. [#15063](https://github.com/ant-design/ant-design/pull/15063)
+- 🌟 Select to supports `showArrow` in multi-select mode. [#15091](https://github.com/ant-design/ant-design/pull/15091)
```
Comment (afc163) [inline CHANGELOG.en-US.md]:
```diff
to supports
supports
```

File: CHANGELOG.zh-CN.md (hunk 2 above)
Comment (afc163) [reply CHANGELOG.zh-CN.md]:
```diff
- typescript
+ TypeScript
```

File: CHANGELOG.en-US.md (hunk 17 above)
Comment (zombieJ) [reply CHANGELOG.en-US.md]: 看着总是怪怪的……

File: CHANGELOG.en-US.md [hunk 23]
```diff
@@ -14,6 +14,28 @@ timeline: true
...
 
 ---
+## 3.14.0
+
+`2019-03-02`
+
+- Two new components added this month:
+  - 🔥🔥🔥[Typography](https://github.com/ant-design/ant-design/pull/14250) provides basic formatting and common operations for text.
+  - 🔥🔥🔥[PageHeader](https://github.com/ant-design/ant-design/pull/13637) can be used to declare the page theme, display important information about the page that the user is interested in, and host the relevant page. Action item.
+- 🌟 TimePicker provides `clearIcon` prop for customizing clear icons. [#14556](https://github.com/ant-design/ant-design/pull/14556)
```
Comment (zombieJ) [inline CHANGELOG.en-US.md]: icons -> icon

File: CHANGELOG.en-US.md [hunk 24]
```diff
@@ -14,6 +14,28 @@ timeline: true
...
+- 🌟 Statistic.Countdown supports `onFinish` prop. [#14791](https://github.com/ant-design/ant-design/pull/14791)
+- 🌟 Support `extra` for Collapse.Panel. [62e65d](https://github.com/ant-design/ant-design/commit/62e65d955065b1862240f9f30d84de44349a0cf9)
```
Comment (zombieJ) [inline CHANGELOG.en-US.md]: 和上面一致，xxx support xxx prop

File: CHANGELOG.en-US.md [hunk 25]
```diff
@@ -14,6 +14,28 @@ timeline: true
...
+- 🌟 Support goto button in Pagination.  [#14819](https://github.com/ant-design/ant-design/pull/14819)
```
Comment (zombieJ) [inline CHANGELOG.en-US.md]: 看起来是 TS 定义更新，是不是应该移下去？

File: CHANGELOG.en-US.md [hunk 26]
```diff
@@ -14,6 +14,28 @@ timeline: true
...
+- 🐞 Fix the typescript type problem for Table `Column Filter`. [#15056](https://github.com/ant-design/ant-design/pull/15056)
```
Comment (zombieJ) [inline CHANGELOG.en-US.md]: Typescript

File: CHANGELOG.en-US.md [hunk 27]
```diff
@@ -14,6 +14,28 @@ timeline: true
...
+- DatePicker
+  - 🐞 Fix DatePicker not support name prop. [#15029](https://github.com/ant-design/ant-design/pull/15029)
+  - 🐞 Fix missing problems with DatePicker `onPanelChange` typescript. [#15043](https://github.com/ant-design/ant-design/pull/15043)
```
Comment (zombieJ) [inline CHANGELOG.en-US.md]: Typescript

File: CHANGELOG.en-US.md (hunk 26 above)
Comment (afc163) [reply CHANGELOG.en-US.md]: TypeScript

Comment (zombieJ) [reply CHANGELOG.en-US.md]: Typescript 相关的统一放到末尾一类去：https://github.com/ant-design/ant-design/releases/tag/3.13.5

File: CHANGELOG.en-US.md [hunk 28]
```diff
@@ -14,6 +14,28 @@ timeline: true
...
+- Two new components added this month:
+  - 🔥🔥🔥[Typography](https://github.com/ant-design/ant-design/pull/14250) provides basic formatting and common operations for text.
+  - 🔥🔥🔥[PageHeader](https://github.com/ant-design/ant-design/pull/13637) can be used to declare the page theme, display important information about the page that the user is interested in, and host the relevant page. Action item.
+- 🌟 TimePicker provides `clearIcon` prop for customizing clear icons. [#14556](https://github.com/ant-design/ant-design/pull/14556)
+- 🌟 Statistic.Countdown supports `onFinish` prop. [#14791](https://github.com/ant-design/ant-design/pull/14791)
+- 🌟 Support `extra` for Collapse.Panel. [62e65d](https://github.com/ant-design/ant-design/commit/62e65d955065b1862240f9f30d84de44349a0cf9)
+- 🌟 Support goto button in Pagination.  [#14819](https://github.com/ant-design/ant-design/pull/14819)
+- 🐞 Fix the typescript type problem for Table `Column Filter`. [#15056](https://github.com/ant-design/ant-design/pull/15056)
+- DatePicker
+  - 🐞 Fix DatePicker not support name prop. [#15029](https://github.com/ant-design/ant-design/pull/15029)
```
Comment (zombieJ) [inline CHANGELOG.en-US.md]: 标题里有 DatePicker 后，下面的 sub item 不需要再写 DatePicker 了

**Review by zombieJ — APPROVED** [review]

**Comment by afc163** [comment]
- [ ] https://github.com/ant-design/ant-design/commit/48789924009ee3c563e3cc7f7c51f364f77ce376
- [ ] https://github.com/ant-design/ant-design/commit/71a774b16bff0f5d802ce988397a8adb21b03fa3
- [ ] https://github.com/ant-design/ant-design/commit/29647f073a212a5088ea7715183ba8d16d78bd97
- [ ] https://github.com/ant-design/ant-design/commit/66786c7c4b6ff2af580826f1cd3a2137be436121
- [ ] https://github.com/ant-design/ant-design/commit/7bbeaff3ed5d00b1f9eb9673d7119af8444c5594
- [ ] https://github.com/ant-design/ant-design/commit/924e8e4b83886a42f8c0295d989e1e9275a656f5
- [ ] https://github.com/ant-design/ant-design/commit/f66555408076c8a08b77dabee305e49230101638

**Comment by chenshuai2144** [comment]
https://github.com/ant-design/ant-design/commit/29647f073a212a5088ea7715183ba8d16d78bd97 
无需体现吧，用户没有任何感知

**Comment by afc163** [comment]
相当于新增了两个 less 变量。

**Pull request merged**